src/wp-settings.php

```diff
@@ -320,6 +320,8 @@
 require ABSPATH . WPINC . '/class-wp-block.php';
 require ABSPATH . WPINC . '/class-wp-block-list.php';
 require ABSPATH . WPINC . '/class-wp-block-parser.php';
+require ABSPATH . WPINC . '/class-wp-block-parser-block.php';
+require ABSPATH . WPINC . '/class-wp-block-parser-frame.php';
```

Comment:
Is this order correct ? In the original file, the split-off classes were "loaded" before the `WP_Block_Parser` class, now they are loaded _after_. (not that it should make much difference, but just making sure it won't be an issue).

Reply:
I don't think it would make any difference, but just to be safe, I pushed an additional commit to preserve their original order 👍